wallet/txauthor/author.go

```diff
@@ -70,102 +73,140 @@ type ChangeSource struct {
 	ScriptSize int
 }
 
-// NewUnsignedTransaction creates an unsigned transaction paying to one or more
+// NewUnsignedTransaction creates an unsigned transaction paying to non or more
```

Comment:
Yes, because its also used to sweep the wallet for the btcd/btcctl usecase using `nil`for the outputs, see: [sweep](https://github.com/btcsuite/btcwallet/blob/ca5c26e2557203a5e7d633593495cf2cff1e1963/cmd/sweepaccount/main.go#L282)